vep/__init__.py

```diff
@@ -142,13 +163,27 @@ def run(self):
         # if there is a requirements.pip, go ahead and install all the things
         if os.path.isfile(self.args.pip_requirements):
             print "installing requirements"
-            print target_pip('install', '-r', self.args.pip_requirements, '-I')
+            # installing requirements can take a spell, print output line-wise
+            target_pip('install', '-r', self.args.pip_requirements, '-I', _out=print_line)
```

Comment:
Sadly: 

    $ python
    Python 2.7.10 (default, Jul 14 2015, 19:46:27) 
    [GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.39)] on darwin
    Type "help", "copyright", "credits" or "license" for more information.
    >>> import sh
    >>> ls = sh.Command("ls")
    >>> ls(_out=print)
      File "<stdin>", line 1
        ls(_out=print)
                    ^
    SyntaxError: invalid syntax
    >>> 

Hence the brain-injuring wrapper.